docs/build_from_source.md

```diff
@@ -172,7 +147,51 @@ When set to `0`, OpenVINO&trade Model Server will be built with [MediaPipe](medi
 
 Example:
 ```bash
-make docker_build MEDIAPIPE_DISABLE=0
+make release_image MEDIAPIPE_DISABLE=1
+```
+
+### `PYTHON_DISABLE`
+
+When set to `0`, OpenVINO&trade Model Server will be built with [Python Nodes](python_support/quickstart.md) support. Default value: `0`. 
```

Comment:
Can we say what happens when set to `1`? It feels more natural imho,  since it's basically a flag. 
We could do the same for `MEDIAPIPE_DISABLE`